docker/docker/src/main/java/org/arquillian/cube/docker/impl/util/GitHubUtil.java

```diff
@@ -0,0 +1,36 @@
+package org.arquillian.cube.docker.impl.util;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+
+import org.arquillian.cube.docker.impl.model.LatestRepository;
+import org.glassfish.jersey.client.ClientConfig;
+import com.fasterxml.jackson.jaxrs.json.JacksonJsonProvider;
+
+public class GitHubUtil {
+
+	private static final String DOCKER_MACHINE_LATEST_URL = "https://api.github.com/repos/docker/machine/releases/latest";
+
+	private static Client getClient() {
+		ClientConfig clientConfig = new ClientConfig();
+		clientConfig.register(JacksonJsonProvider.class);
+		return ClientBuilder.newClient(clientConfig);
+	}
+
+	public static String getDockerMachineLatestVersion() {
+		Client client = getClient();
```

Comment:
Yes I know we could use it but since it is only one request and it should be fast to achieve it without using JAXRS client I think it is worth to do it.